app/src/main/res/layout/profile_chooser_add_view.xml

```diff
@@ -57,25 +70,19 @@
 
         <TextView
           android:id="@+id/add_profile_text"
-          android:layout_width="wrap_content"
-          android:layout_height="wrap_content"
-          android:fontFamily="sans-serif-medium"
+          style="@style/Caption"
           android:text="@{hasProfileEverBeenAddedValue ? @string/profile_chooser_add : @string/set_up_multiple_profiles}"
-          android:textColor="@color/white"
-          android:textSize="14sp" />
+          android:textColor="@color/white" />
 
         <TextView
           android:id="@+id/add_profile_description_text"
-          android:layout_width="wrap_content"
-          android:layout_height="wrap_content"
+          style="@style/TextFieldLabel"
           android:layout_gravity="center"
           android:layout_marginTop="4dp"
-          android:fontFamily="sans-serif"
           android:text="@string/set_up_multiple_profiles_description"
           android:textColor="@color/white"
-          android:textSize="12sp"
           android:visibility="@{hasProfileEverBeenAddedValue ? View.GONE : View.VISIBLE}" />
       </LinearLayout>
     </LinearLayout>
-  </LinearLayout>
-</layout>
+  </androidx.constraintlayout.widget.ConstraintLayout>
+</layout>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Done , Pardon me for the delay